app/assess/data.py

```diff
@@ -128,20 +138,34 @@ def get_round_with_applications(
     return None
 
 
+def get_score_and_justification(application_id, sub_criteria_id):
+    url = Config.ASSESSMENT_SCORES_ENDPOINT
+    params = {
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+    }
+    response_json = get_data(url, params)
+    current_app.logger.info(
+        f"Response from Assessment Store: '{response_json}'."
+    )
+    return response_json
+
+
 def submit_score_and_justification(
-    assessment_id, person_id, score, justification, sub_crit_id
+    score, justification, application_id, user_id, sub_criteria_id
 ):
-
     data_dict = {
         "score": score,
         "justification": justification,
-        "person_id": person_id,
+        "user_id": user_id,
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
     }
-    url = Config.ASSESSMENT_SCORES_ENDPOINT.format(
-        assessment_id=assessment_id, sub_criteria_id=sub_crit_id
-    )
+    url = Config.ASSESSMENT_SCORES_ENDPOINT
     response = requests.post(url, json=data_dict)
-    print(response.content)
+    current_app.logger.info(
+        f"Response from Assessment Store: '{response.json}'."
```

Comment:
`json` is a method not a property

```suggestion
        f"Response from Assessment Store: '{response.json()}'."
```